Lib/fontTools/merge.py

```diff
@@ -780,6 +849,7 @@ def merge(self, fontfiles):
 		# TODO Is it necessary to reload font?  I think it is.  At least
 		# it's safer, in case tables were loaded to provide glyph names.
 		fonts = [ttLib.TTFont(fontfile) for fontfile in fontfiles]
+		self.fonts = copy.deepcopy(fonts)
```

Comment:
Re-opening the fonts is ok. But I don't see why you need to store the result in `self.fonts` if you are not going to use it anywhere (at least I don't see it).